storybook/src/data/documents.data.js

```diff
@@ -0,0 +1,66 @@
+import uuid from 'uuid/v4';
+import image from './nuxeo-logo.jpg';
+
+export default class Document {
+  // class methods
+  constructor(type = '', properties = {}, facets = [], permissions = [], isTrashed = false, isFavorite = false) {
+    this.type = type;
+    this.properties = properties;
+    this.facets = facets;
+    this.permissions = permissions;
+    this.isTrashed = isTrashed;
+    this.isFavorite = isFavorite;
+  }
+
+  trash() {
+    this.isTrashed = true;
+    return this;
+  }
+
+  favorite() {
+    this.isFavorite = true;
+    return this;
+  }
+
+  addType(documentType) {
+    this.type = documentType;
+    return this;
+  }
+
+  addPermissions(permission) {
+    this.permissions = permission;
+    return this;
+  }
+
+  addImage() {
```

Comment:
What if we want to set another image? Perhaps something like:
```suggestion
image(content = {name: 'Nuxeo Logo', data: image}) {
```